packages/sonarwhal/src/lib/utils/misc.ts

```diff
@@ -128,20 +136,19 @@ const normalizeString = (value: string, defaultValue?: string): string => {
     return value.toLowerCase().trim();
 };
 
+const protocolRegex = /([^:]*):.*/;
```

Comment:
This also matches `:test`, shouldn't it be `/([^:]+):.*/`?

Reply:
`isRegularProtocol(':test')` should return `true` or `false`?

Reply:
With the current regex it will return `false` because the protocol is the empty string.

Reply:
IMHO `:test` shouldn't match.

Reply:
Yes, nevermind, let's just readd [this comment](https://github.com/sonarwhal/sonarwhal/pull/903/files#diff-99cb63c80e7850a78eac3f222eac6898L139) back.